src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+spring.application.name=ak
```

Comment:
This is application.properties.

Google if we can create different property file for different environments as well so that we can have separate config for each env (dev/stage/prod).

If yes create for all env.

Reply:
Done